events/events.go

```diff
@@ -58,20 +58,22 @@ func routeEvent(msg *events.Envelope) {
 	}
 }
 
-func SetupEventRouting(wantedEvents string) {
+func SetupEventRouting(wantedEvents string) error {
 	selectedEvents = make(map[string]bool)
-	for _, event := range strings.Split(wantedEvents, ",") {
-		if isAuthorizedEvent(event) {
-			selectedEvents[event] = true
-			log.LogStd(fmt.Sprintf("Event Type [%s] is included in the fireshose!", event), false)
-		} else {
-			log.LogError(fmt.Sprintf("Rejected Event Name [%s] - See wanted/selected events arg. Check Your Command Line Arguments!", event), "")
-		}
-	}
-	// If any event is not authorize we fallback to the default one
-	if len(selectedEvents) < 1 {
```

Comment:
No pb dude, let' say I put --events=badEventName what's happen ?
selectedEvents is empty ... so we should fallback to LogMessage.